HelpSource/Classes/Pclump.schelp

```diff
@@ -6,7 +6,7 @@ categories:: Streams-Patterns-Events>Patterns>List
 DESCRIPTION::
 Groups the source pattern into arrays whose size is given by n. Similar to LINK::Classes/SequenceableCollection#-clump::.
 
-E.g. If the source pattern has 5 elements and you choose a clump size of 2, the new pattern will return two arrays with containing 2 elements, and a final array containing 1 element.
+E.g. If the source pattern has 5 elements and you choose a clump size of 2, the new pattern will return two array containing 2 elements and a final array containing 1 element.
 
```

Comment:
two arrays?

Reply:
Yeah. Say you have Pclump(2, Pseq([1,2,3,4,5]) - you'll effectively get the sequence:
Pseq([1,2], [3,4], [5])

I'm trying to make this as explicit as possible, as I find it a little frustrating having to work out this kind of behavior for patterns. It's easy to imagine a different pattern which chopped off the final array (effectively Pseq([1,2], [3,4])

Reply:
Maybe I should just rewrite what I said above. Something along the lines of 
Pclump(2, Pseq([1,2,3,4,5])) == Pseq([1,2], [3,4], 5)